app/src/main/java/com/jerboa/ui/components/settings/lookandfeel/LookAndFeelActivity.kt

```diff
@@ -107,6 +117,27 @@ fun LookAndFeelActivity(
                     .verticalScroll(scrollState)
                     .padding(padding),
             ) {
+                SettingsListDropdown(
+                    title = {
+                        Text(text = "Select your language")
```

Comment:
This string should be localized, but maybe just "Language" for the English version.